egs2/chime7_task1/asr1/local/install_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[ -f ./path.sh ] && . ./path.sh
+
+# install lhotse from master, we need the most up-to-date one
+pip install git+https://github.com/lhotse-speech/lhotse
+
+#check if kaldi has been installed and compiled
+if ! command -v wav-reverberate &>/dev/null; then
+  echo "It seems that wav-reverberate Kaldi command cannot be found.
+  This happens if you don't have compiled and installed Kaldi.
+  Please follow instructions in ${MAIN_ROOT}/tools/kaldi/INSTALL. "
+fi
+
+# install s3prl
+./tools/installers/install_s3prl.sh
+
+if ! command -v gss &>/dev/null; then
+  conda install -yc conda-forge cupy=10.2
+  ./${MAIN_ROOT}/tools/install_gss.sh
+fi
+
+sox_conda=`command -v ../../../tools/venv/bin/sox 2>/dev/null`
```

Comment:
As well as the previous comment, we can't use the path of `../../../tools/venv/bin/sox`.

Please just use `sox` command.

Reply:
https://github.com/espnet/espnet/blob/197e4f68695f1009c69e2bd7ea3c681d73df3a95/egs2/chime7_task1/asr1/local/generate_chime6_data.sh#L59 I was mainly following this script (same as Kaldi recipe) that is required for generating CHiME-6 from CHiME-5

Reply:
can we made an exception here and force sox usage through conda ? (because the recipe needs conda)

Reply:
Is there a reason using conda-sox? I think the requirement is using sox-v14.4.2.

Reply:
IDK but this is what the Kaldi script did. I think original author was @sw005320 . We can change it BTW.

Reply:
Ok thank you ! 
I think it is better we keep it that way then and raise an error if conda is not found. 
I would like that the data is reproducible as we are using this hash for participants to check if the data preparation is consistent among all: 
https://github.com/espnet/espnet/blob/7cf99cf9120e2ea7e3fb47b598ae93284806751b/egs2/chime7_task1/asr1/README.md?plain=1#L182